bundles/org.eclipse.ui/schema/perspectiveExtensions.exsd

```diff
@@ -88,6 +89,27 @@ are appended to the initial definition.
                </restriction>
             </simpleType>
          </attribute>
+         <attribute name="editorOnboardingText" type="string">
+            <annotation>
+               <documentation>
+                  The text is shown in the editor area in case no editor is open.
+               </documentation>
+               <appInfo>
+                  <meta.attribute translatable="true"/>
+               </appInfo>
+            </annotation>
+         </attribute>
+         <attribute name="editorOnboardingImage" type="string">
+            <annotation>
+               <documentation>
+                  The image is shown in the editor area in case no editor is open.
```

Comment:
Maybe we should also say how big (in pixels) the icon should be.

Reply:
VS code has 280x280. I like it with this size.
I found the SVG in https://github.com/eclipse-platform/eclipse.platform.images and I have created a grey images with 280x280.

I think I don't need to craete the svg as it is alraedy in https://github.com/eclipse-platform/eclipse.platform.images. 
Do I need a second png? with 560x560 then?

Reply:
added images in 350x350 and double size.